src/components/tables/cells/ChipList/index.tsx

```diff
@@ -7,9 +7,10 @@ interface Props {
     strings: string[];
     disabled?: boolean;
     maxChips?: number;
+    stripPath?: boolean;
 }
 
-function ChipList({ strings, disabled, maxChips }: Props) {
+function ChipList({ strings, disabled, maxChips, stripPath }: Props) {
```

Comment:
We don't want to always strip the pathing so we can use chips for things other than entity names